docs/source/devstack.rst

```diff
@@ -0,0 +1,200 @@
+
+Steps
+-----
+
+1. Update the Vagrantfile 
+
+You need to update the Vagrantfile to provide access to port 3000 from the VM to the host (your local system)
+
+Edit your ``<project>/Vagrantfile``
+
+Add the following line to the conditional block ``if not ENV['VAGRANT_NO_PORTS']``::
+
+	config.vm.network :forwarded_port, guest: 3000, host: 3000  # Figures webpack server
+
+
+After updating, this section should look like::
+
+	  # If you want to run the box but don't need network ports, set VAGRANT_NO_PORTS=1.
+	  # This is useful if you want to run more than one box at once.
+	  if not ENV['VAGRANT_NO_PORTS']
+	    config.vm.network :forwarded_port, guest: 8000, host: 8000  # LMS
+	    config.vm.network :forwarded_port, guest: 8001, host: 8001  # Studio
+	    config.vm.network :forwarded_port, guest: 8002, host: 8002  # Ecommerce
+	    config.vm.network :forwarded_port, guest: 8003, host: 8003  # LMS for Bok Choy
+	    config.vm.network :forwarded_port, guest: 8031, host: 8031  # Studio for Bok Choy
+	    config.vm.network :forwarded_port, guest: 8120, host: 8120  # edX Notes Service
+	    config.vm.network :forwarded_port, guest: 8765, host: 8765
+	    config.vm.network :forwarded_port, guest: 9200, host: 9200  # Elasticsearch
+	    config.vm.network :forwarded_port, guest: 18080, host: 18080  # Forums
+	    config.vm.network :forwarded_port, guest: 8100, host: 8100  # Analytics Data API
+	    config.vm.network :forwarded_port, guest: 8110, host: 8110  # Insights
+	    config.vm.network :forwarded_port, guest: 9876, host: 9876  # ORA2 Karma tests
+	    config.vm.network :forwarded_port, guest: 50070, host: 50070  # HDFS Admin UI
+	    config.vm.network :forwarded_port, guest: 8088, host: 8088  # Hadoop Resource Manager
+	    config.vm.network :forwarded_port, guest: 18381, host: 18381    # Course discovery
+	    config.vm.network :forwarded_port, guest: 3000, host: 3000 # Figures webpack server
+	  end
+
+2. Clone the Figures repository
+
+On your host, go to the ``<project>/src`` folder and run::
+
+	git clone git@github.com:appsembler/figures.git
+
+3. Install the Figures Python package
+
+From your host's shell, ssh to the Vagrant VM::
+
+	vagrant ssh
+
+Change to the *edxapp* user::
+
+	sudo su edxapp
+
+Navigate to the Figures source directory::
+
+	cd /edx/src/figures
+
+Install the package with pip::
+
+	pip install -e .
+
+Figures should now be installed. To confirm, run the following::
+
+	pip list | grep figures
+
+You should see a line like::
+
+	Figures (0.1.0, /edx/src/figures)
+
+
+4. Update the Devstack LMS env file
+
+Continuing as user *edxapp*, edit the ``/edx/app/edxapp/lms.env.json`` file.
+
+As a top level key, add the following::
+
+	"ADDL_INSTALLED_APPS": [
+		"figures"
+	]
+
+We suggest adding the above immediately after ``ACTIVATION_EMAIL_SUPPORT_LINL`` so that it is in alphabetical order.
```

Comment:
``ACTIVATION_EMAIL_SUPPORT_LINL` -> `ACTIVATION_EMAIL_SUPPORT_LINK`

Reply:
Fixed. Thanks for finding the typo